index.html

```diff
@@ -293,8 +296,35 @@ <h3 class="blue-fg">LET'S CONNECT</h3>
                     <!-- Get My Resumee button (FLEX vertical) -->
                     <div class="flex-box horizontal flex-start">
                         <button class="works-card-button">Get My Resumee</button>
-                    </div>                    
+                    </div>
+                    
+                    
+
                 </div>
+
+                <!-- Technologies Accordion -->
+                <dl class="tech-menu">
+                    <dt>Languages</dt>
+                    <dd>
+                        <ul>
+                            <li>
+                                <img src="@images/JS_logo.svg" alt="JS logo">
+                                JavaScript
+                            </li>
+                            <li>
+                                <img src="@images/HTML-5_logo.svg" alt="HTML 5 logo">
+                                HTML
+                            </li>
+                            <li>
+                                <img src="@images/CSS-3_logo.svg" alt="CSS 3 logo">
+                                CSS
+                            </li>
+                        </ul>
+                    </dd>
+                    <dt class="border-bottom">Frameworks</dt>
+                    <dt class="border-bottom">Skills</dt>
```

Comment:
- [ ] Your technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li` tag for best semantic practices.
- [ ] Also your technology categories are missing some icons. It would be good to add them so as to follow the Figma design.
- Your design:
![Annotation 2022-02-20 070309](https://user-images.githubusercontent.com/85113567/154831484-d19dc466-f174-43a5-9acd-fdc904056088.jpg)
- Figma design:
![Annotation 2022-02-20 070329](https://user-images.githubusercontent.com/85113567/154831487-6cc1ee99-6644-49d6-9745-b6f6e2798f55.jpg)